src/AtlasAdmin.ts

```diff
@@ -0,0 +1,172 @@
+// @ts-ignore
+import Logger from 'basic-logger';
+import { request, RequestOptions } from 'urllib';
+import { SearchIndex } from './data/atlas-search-index';
+import { SearchIndexResponse, IndexMappings } from './data/atlas-types';
+import { Taxonomy } from './SearchIndex';
+
+const DEFAULT_ATLAS_API_OPTIONS: RequestOptions = {
+  headers: {
+    'content-type': 'application/json',
+  },
+  dataType: 'json',
+};
+
+const log = new Logger({
+  showTimestamp: true,
+});
+
+const CLUSTER_NAME = process.env['CLUSTER_NAME'] || 'Search';
+const COLLECTION_NAME = process.env['COLLECTION_NAME'] || 'documents';
+const GROUP_ID = process.env['GROUP_ID'] || '';
+const DB = process.env['ATLAS_DATABASE'] || 'search';
+const SEARCH_INDEX = 'default';
+
+/**
+ * Manager is intended to keep Atlas Search Index in sync across environments
+ * Index should be edited in ../data directory and expect to be updated on deploy
+ */
+
+export class AtlasAdminManager {
+  private defaultHeaders: RequestOptions;
+  private baseUrl: string;
```

Comment:
Super NIT: Would it be worth while making `baseUrl` readonly?

Reply:
done